libvast/native-plugins/rest_api/export.cpp

```diff
@@ -0,0 +1,237 @@
+//    _   _____   __________
```

Comment:
Would it maybe make more sense to bundle these native plugins as part of the REST plugin itself? I think that can be implemented in our CMake with some shuffling around.

The URL should probably also match plugin name, i.e., `rest_endpoints` rather than `rest_api`.

Reply:
Either way makes sense to me; I mainly split them out of the rest plugin so I don't accidentally rely on internals of the REST plugin in the implementation, which would lead to future problems for closed-source endpoints.

Reply:
I did the renaming, I didn't do the reshuffling for now since at least then we have all native plugins in one place.